tcf_website/views/review.py

```diff
@@ -123,3 +86,23 @@ def get_object(self):  # pylint: disable=arguments-differ
             raise PermissionDenied(
                 "You are not allowed to delete this review!")
         return obj
+
+
+@login_required
+def edit_review(request, review_id):
+    """Review modification view."""
+    review = get_object_or_404(Review, pk=review_id)
+    if review.user != request.user:
+        raise PermissionDenied('You are not allowed to edit this review!')
+
+    if request.method == 'POST':
+        form = ReviewForm(request.POST, instance=review)
+        if form.is_valid():
+            form.save()
+            messages.success(request,
+                             f'Successfully updated the review for {form.instance.course}!')
```

Comment:
Maybe should be "updated **your** review"?

Reply:
Fixed in cbd75728f4dfd61dde70533958f9143247dc6ead